### ISSUE TITLE
Is there a way to customise colors?

### ISSUE THREAD

**Issue by johnnyreilly** [litixsoft/karma-mocha-reporter#30]
Hi,

First of all, thanks for karma mocha reporter - love it!

My question is: is there a way to customise the colors used?  Whilst the white is appropriately white on Windows, the green and red for pass and fail are really hard to read.  

If there was a way to control this that would be fantastic.

**Comment by 4kochi** [comment]
Hi John,

nice that your are find the module useful. For printing the colors we use calk (https://github.com/chalk/chalk#colors).

We can add an option to the config in which you can change the color, but we can only use the colors an bgColors that chalk exposes. It might look like the following: 

```javascript
mochaReporter: {
    colors: {
        success: 'bgGreen',
        warning: 'yellow',
        error: 'cyan'
    }
}
```

Please let us know if that would be helpful for you.

**Comment by johnnyreilly** [comment]
I'm familiar with Chalk.  The config option sounds great!  

I use `bgRed` elsewhere to alert me when my browserify build fails (which is plenty legible) and so I can see myself using that for errors.

**Comment by johnnyreilly** [comment]
You're kidding?! That's fast!!! :+1: I'll give this a try in the morning - thanks!

**Comment by 4kochi** [comment]
No problem, was an easy task :smile: 

Just released to npm as v1.1.0. The travis test are broken, but it works locally. Seems to be a problem with karma in general.

**Comment by jcrben** [comment]
How about passing in options such as dim or bold?
